packages/grub2/config/bootargs.cfg.tmpl

```diff
@@ -2,7 +2,7 @@ set kernel=/boot/vmlinuz
 if [ -n "$recoverylabel" ]; then
     set kernelcmd="console=tty1 console=ttyS0 root=live:LABEL=$recoverylabel rd.live.dir=/ rd.live.squashimg=$img panic=5 rd.neednet=1 rd.cos.oemlabel=@OEM_LABEL@"
 else
-    set kernelcmd="console=tty1 console=ttyS0 root=LABEL=$label cos-img/filename=$img panic=5 security=selinux selinux=1 rd.neednet=1 rd.cos.oemlabel=@OEM_LABEL@"
+    set kernelcmd="console=tty1 console=ttyS0 root=LABEL=$label cos-img/filename=$img panic=5 security=selinux selinux=1 rd.neednet=1 rd.cos.oemlabel=@OEM_LABEL@ fsck.mode=force fsck.repair=yes"
```

Comment:
I don't see any `fsck.repair` definition, hence I guess this is something that  `systemd-fsck` parses, right? If this something custom from our side I'd suggest defining it with the `rd.cos...` prefix, I think it is not the case though, just to double check.

Reply:
yup, those are read by systemd-fsck directly, so we don't need to do any special handling ourselves :partying_face: